src/lib/gadgets/gadgets.ts

```diff
@@ -39,6 +40,16 @@ const Gadgets = {
   rangeCheck64(x: Field) {
     return rangeCheck64(x);
   },
+
+  /**
+   * Asserts that the input value is in the range [0, 2^8).
+   *
+   * See {@link Gadgets.rangeCheck8} for analogous details and usage examples.
```

Comment:
this points to the same Gadget - did you mean `Gadgets.rangeCheck32`?

Reply:
ah I meant `rangeCheck64` thanks - stupid copilot 🤖